<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote PW Code Gen V1.1</string>
  </property>
  <widget class="QLabel" name="Lbl_Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="Frm_Password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Password1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Password">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Password2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ReEnter Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Ok">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Cancel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SecretCode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SecretCode">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Secret Code</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Frm_Main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Btn_Exit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_CreateCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ChangePassword">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ChangeEncryptionPassword">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Encyption PW</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Chk_ShowPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Frm_PasswordEnc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_PasswordEnc">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_PasswordEnc">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Encryption PW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_OkEnc">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ExitEnc">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
